drivers/adc/adc_lmp90xxx.c

```diff
@@ -110,7 +111,7 @@ struct lmp90xxx_config {
 	u8_t spi_cs_pin;
 	struct spi_config spi_cfg;
 	const char *drdyb_dev_name;
-	u32_t drdyb_pin;
+	gpio_pin_t drdyb_pin;
 	int drdyb_flags;
```

Comment:
This should be `gpio_dt_flags_t`.

Reply:
If you make it `gpio_devicetree_flags_t` it'll get converted when the rename occurs.

Reply:
@pabigot Thanks, changed.